src/app/application/components/app-view/AppView.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <MainView
```

Comment:
I don't really want to change this now but just realized this probably shouldn't be called `SomethingView.vue` as its not a View/Route component. If someone felt strongly that I should do it now I'd be happy to do it here, otherwise it can be done at somepoint in the future.

Reply:
Fine by me.